src/shared_modules/utils/rocksDBWrapper.hpp

```diff
@@ -56,6 +59,39 @@ namespace Utils
         {
         }
 
+        /**
+         * @brief Class destructor. Frees column family handles.
+         *
+         */
+        ~RocksDBWrapper()
+        {
+            std::for_each(m_handles.begin(),
+                          m_handles.end(),
+                          [this](rocksdb::ColumnFamilyHandle* handle) { m_db->DestroyColumnFamilyHandle(handle); });
+        }
+
+        /**
+         * @brief Creates a new column family in the database.
+         *
+         * @param columnName Name of the new column.
+         */
+        void createColumn(const std::string& columnName)
+        {
+            if (columnName.empty())
+            {
+                throw std::runtime_error {"Column name is empty"};
```

Comment:
Improve: To align with the other methods
```suggestion
                throw std::invalid_argument {"Column name is empty"};
```

Reply:
Done!